unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PlayerAvatarController/PlayerAvatarController.cs

```diff
@@ -228,20 +248,40 @@ private async UniTaskVoid LoadingAvatarRoutine(UserProfile profile, Cancellation
             IAvatarAnchorPoints anchorPoints = new AvatarAnchorPoints();
             anchorPoints.Prepare(avatarContainer.transform, avatar.GetBones(), AvatarSystemUtils.AVATAR_Y_OFFSET + avatar.extents.y);
 
-            var player = new Player()
+            var player = new Player
             {
                 id = userProfile.userId,
                 name = userProfile.name,
                 avatar = avatar,
                 anchorPoints = anchorPoints,
                 collider = avatarCollider
             };
+
             DataStore.i.player.ownPlayer.Set(player);
 
             enableCameraCheck = true;
             avatarCollider.gameObject.SetActive(true);
             CommonScriptableObjects.rendererState.RemoveLock(this);
             DataStore.i.common.isPlayerRendererLoaded.Set(true);
+
+            var entity = new DecentralandEntity
```

Comment:
I have taken a look below and it seems like it's almost ready to be compatible with null entities, we are only missing support in `CollidersManager.AddOrUpdateEntityCollider`. 

Can you try adding it there and avoid this entity? I'm approving regardless because maybe you will find more instances of usage of this entity down the rabbit hole that Im missing, if so, we can leave it as it is.

Reply:
I've uploaded the support for null entities. However the change could be kind of risky. I did find 46 usages of the entity. Many cases supports nulls, but there are corner cases that do not. On this particular case, i think it will not bring major issues, since only affects the pointer events. Will wait for QA's approval, so we dont introduce any major issue